backend/samfundet/models/recruitment.py

```diff
@@ -201,3 +201,21 @@ def save(self, *args: tuple, **kwargs: dict) -> None:
                 self.interview = Interview.objects.create()
 
         super().save(*args, **kwargs)
+
+
+class OccupiedTimeslots(FullCleanSaveMixin):
+
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        null=False,
+        blank=False,
+        help_text='Occupied timeslots for user',
+        related_name='occupied_timeslots',
+    )
+    # Mostly only used for deletion, and anonomization
+    recruitment = models.ForeignKey(Recruitment, on_delete=models.CASCADE, help_text='Occupied timeslots for the users for this recruitment')
+
+    # Start and end time of availability
+    start_time = models.DateTimeField(help_text='The time of the interview', null=False, blank=False)
+    end_time = models.DateTimeField(help_text='The time of the interview', null=False, blank=False)
```

Comment:
Litt enklere å vite hva feltet inneholder.
```suggestion
    start_dt = models.DateTimeField(help_text='The time of the interview', null=False, blank=False)
    end_dt = models.DateTimeField(help_text='The time of the interview', null=False, blank=False)
```

Reply:
Done